.github/CODEOWNERS

```diff
@@ -13,12 +13,6 @@
 /*.md                                         @vercel/next-js @leerob
 /packages/create-next-app/                    @vercel/next-js
 
-# Tooling & Telemetry
-
-/packages/next/src/build/                     @timneutkens @ijjk @shuding @vercel/web-tooling @huozhi
-/packages/next/src/telemetry/                 @timneutkens @ijjk @shuding @padmaia
-/packages/next-swc/                           @timneutkens @ijjk @shuding @vercel/web-tooling
-
 # Image Component (@styfle)
 
 /**/*image*                                   @timneutkens @ijjk @shuding @styfle
```

Comment:
unfortunately, these 2 image wildcard lines override our ownership in some cases